observability/opentelemetry/src/main/resources/org/wildfly/extension/opentelemetry/LocalDescriptions.properties

```diff
@@ -0,0 +1,13 @@
+opentelemetry=OpenTelemetry subsystem
+opentelemetry.add=Operation Adds subsystem
+opentelemetry.batch-delay=The interval, in milliseconds, between two consecutive exports. Default is 5000.
+opentelemetry.endpoint=The URL for the exporter endpoint.
+opentelemetry.export-timeout=The maximum allowed time, in milliseconds, to export data.
+opentelemetry.exporter-type=The name of the exporter to use
+opentelemetry.max-export-batch-size=The maximum size of the export batch.
+opentelemetry.max-queue-size=The maximum size of the queue before it is exported.
+opentelemetry.remove=Operation Removes subsystem
+opentelemetry.sampler-arg=The argument to use to configure the sampler
+opentelemetry.sampler-type=The sampling strategy to use
+opentelemetry.service-name=Service name
```

Comment:
This is not much helpful. How is this used?

Reply:
Updated.